Include changes for release 1.0.0 (#131)
qbicsoftware/postman-cli: README.md

  
Commands:
  download  Download data from OpenBis

>   list      List all available datasets for the given identifiers and get further information

Reviewer: ```suggestion
  list      List all available datasets for the given identifiers with additional metadata 
```
